ecom_test/tests/backend_tests/products/test_list_products_with_filter.py

```diff
@@ -0,0 +1,40 @@
+
+import pytest
+from ecom_test.src_file.api_helpers.ProductsAPIHelpers import ProductsAPIHelper
+from ecom_test.src_file.dao.products_dao import ProductsDAO
+from datetime import datetime, timedelta
+
+
+@pytest.mark.regression
+class TestListProductsWithFilter(object):
+
+    # create a method to get the list of products with filter for specific period of time from now
+
+    @pytest.mark.tcid35
+    def test_list_products_with_filter_after(self):
+        # Create a payload data request to be sent
+        x_days_from_now = 60  # variable for requesting data for last 30 days form now
+        after_created_date = datetime.now().replace(microsecond=0) - timedelta(days=x_days_from_now)
+        after_date = after_created_date.isoformat()  # ISO8601 date format
+        pyload = dict()  # data will be in dictionary format (Jason format)
+        pyload['after'] = after_date
+        pyload['per_page'] = 100  # showing 100 products per page.
+
+        # make an API call
+
+        rs_api = ProductsAPIHelper().call_list_products(pyload)
+        assert rs_api, f" Empty response for 'list products with filter'"
+
+        # get the data from database
+
+        db_products = ProductsDAO().get_products_created_after_given_date(after_date)
+
+        # verify the responses to match DB
+
```

Comment:
not a fan of the space after the comment. but good job adding comments.

Reply:
extra spaces removed